packages/modelviewer.dev/examples/annotations/index.html

```diff
@@ -312,11 +312,295 @@ <h4 id="intro"><span class="font-medium">Show Dimensions. </span></h4>
+    data-orbit="-37.54149deg 82.16209deg 0.0468692m" data-target="0.08566038m 0.1249514m -0.1939646m" data-fov="3.52deg">
+    Lucky Shot
+  </button>      
+  <button class="view-button" slot="hotspot-7" 
+    data-position="0.14598m 0.03177m -0.05945886m" data-normal="-0.9392524m 0.2397608m -0.2456009m" 
+    data-orbit="-142.3926deg 86.45934deg 0.06213665m" data-target="0.1519967m 0.01904771m -0.05945886m" data-fov="0.06deg">
+    Get Away!
+  </button>      
+  <button class="view-button" slot="hotspot-8" 
+    data-position="0.0094m 0.0894m -0.15103m" data-normal="-0.3878782m 0.4957891m -0.7770094m" 
+    data-orbit="-118.6729deg 117.571deg 0.03905975m" data-target="0.007600758m 0.06771782m -0.1386167m" data-fov="2.93deg">
+    The Jump
+  </button>      
+  <button class="view-button" slot="hotspot-9" 
+    data-position="-0.0658m 0.1786m -0.0183m" data-normal="0.7857152m 0.4059967m 0.46671m" 
+    data-orbit="53.28236deg 95.91318deg 0.1102844m" data-target="-0.07579391m 0.1393538m -0.00851791m" data-fov="8.27deg">
+    The Beast
+  </button>     
+  <button class="view-button" slot="hotspot-10" 
+    data-position="0.02610224m 0.01458751m -0.004978945m" data-normal="-0.602551m 0.7856147m -0.1405055m" 
+    data-orbit="-78.89725deg 77.17752deg 0.08451112m" data-target="0.02610223m 0.0145875m -0.004978945m" data-fov="6.34deg">
+    Treasure
+  </button>
+  <button class="view-button" slot="hotspot-11" 
+    data-position="-0.1053838m 0.01610652m 0.1076345m" data-normal="-0.624763m 0.5176854m 0.5845283m" 
+    data-orbit="10.89188deg 119.9775deg 0.03543022m" data-target="-0.1053838m 0.01610652m 0.1076345m" data-fov="2.66deg">
+    Desperation
+  </button>
+</model-viewer>
+
+<script type="module">
+  const modelViewer = document.querySelector("#hotspot-camera-view-demo");
+  const annotationClicked = (annotation) => {
+    let dataset = annotation.dataset;
+    modelViewer.cameraTarget = dataset.target;
+    modelViewer.cameraOrbit = dataset.orbit;
+    if(dataset.fov !== undefined)
+        modelViewer.fieldOfView = dataset.fov;
+  }
+
+  modelViewer.querySelectorAll('button').forEach((hotspot) => {
+    console.log("hello hotspot", hotspot);
+    hotspot.addEventListener('click', () => annotationClicked(hotspot));
+  });
+</script>
+
+<style type="text/css">
+  .view-button {
+      background: #fff;
+      border-radius: 4px;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25);
+      color: rgba(0, 0, 0, 0.8);
+      display: block;
+      font-family: Futura, Helvetica Neue, sans-serif;
+      font-size: 12px;
+      font-weight: 700;
+      max-width: 128px;
+      overflow-wrap: break-word;
+      padding: 0.5em 1em;
+      position: absolute;
+      width: max-content;
+      height: max-content;
+      transform: translate3d(-50%, -50%, 0);
+    }
+</style>
+
+<!-- nearly identical to the "Staging & Cameras/Panning" example -->
+<script type="module">
```

Comment:
I do like the idea of pulling this script out of the snippet and just referencing the other example in the text. I'm surprised pulling it out didn't work. Any idea what's going wrong?